lib/widget/confirmation_input.dart

```diff
@@ -64,7 +73,11 @@ class ConfirmationInput extends StatefulWidget
 
   @override
   Map<String, Function> methods() {
-    return {};
+    return {
+      'clear': () => controller.inputFieldAction?.clear(),
```

Comment:
you'll be repeating these a lot. Can you do this on the controller itself once?

Reply:
Like this - https://github.com/EnsembleUI/ensemble/pull/765/files#diff-817acf83c5ba49db96f33064a06e666c213c175ddf4e2dbe36795b92aab1281d

But I'm getting method not found exception and couldn't see any getBaseMethods usage in our ensemble. Am I doing wrong? I updated the PR with these changes.

`Error Message: cannot compute statement=CallExpression as no method found for property=clearCode: confirmationInputId.clear()`

Reply:
@vusters any idea about the error above?

Reply:
Not sure, @vinothvino42 you have to look into your logic. Alternatively, have a InputController (if not already exists) that will do these directly without InputFieldAction.

Reply:
```dart
class InputController extends BoxController {
  InputFieldAction? inputFieldAction;

  @override
  Map<String, Function> getBaseMethods() {
    Map<String, Function> methods = super.getBaseMethods();
    methods.addAll({
      'flutter': () => print("Flutter"),
    });
    return methods;
  }

  @override
  Map<String, Function> getBaseSetters() {
    Map<String, Function> setters = super.getBaseSetters();
    setters.addAll({
      'flutterSet': (value) => print("Value: $value"),
    });
    return setters;
  }
}
```

Here, when I assigned the value to the setter and called the methods. setters are working fine but methods are not calling their methods in the controller.
```yaml
Button:
  label: Flutter
  onTap: |
    //@code
    confirmationInputId.flutterSet = 'Apple Developer';
    confirmationInputId.flutter();
```

**Log:**
```
flutter: Value: Apple Developer
flutter: Line: 4 in YAML and Line: 2 within the code block. Error Message: cannot compute statement=CallExpression as no method found for property=flutterCode: confirmationInputId.flutter()
```

**Exception Screenshot:**
![Screenshot 2023-08-31 at 3 41 29 PM (2)](https://github.com/EnsembleUI/ensemble/assets/12869588/52f5c1f3-1be2-4a80-a4f2-5942f70535a9)

Reply:
Revert back to mixin approach